go-controller/pkg/ovn/namespace.go

```diff
@@ -331,8 +331,17 @@ func (oc *Controller) deleteNamespace(ns *kapi.Namespace) {
 		return
 	}
 	defer nsInfo.Unlock()
+
+	klog.V(5).Infof("Deleting Namespaces's NetworkPolicy entities")
+	for _, np := range nsInfo.networkPolicies {
+		delete(nsInfo.networkPolicies, np.name)
+		np.deleted = true
```

Comment:
I think we can move this `np.deleted = true` to cleanNPentities/destroyNamespacePolicy()? That way it only needs to be set in one place. Both places that set it call the clean/destroy function right after.

Reply:
Yep I agree @dcbw should be all set now